### ISSUE TITLE
Creation of Reference Image errors

### ISSUE THREAD

**Issue by Finking4** [NevilArt/BsMax_2_80#11]
Here is the trace

Python: Traceback (most recent call last):
  File "C:\Users\...\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\BsMax_2_80-master\primitive\empty.py", line 68, in execute
    bpy.ops.create.empty('INVOKE_DEFAULT',empty_type="IMAGE",depth="DEFAULT")
  File "C:\Program Files\Blender Foundation\Blender 2.91\2.91\scripts\modules\bpy\ops.py", line 130, in __call__
    ret = _op_call(self.idname_py(), C_dict, kw, C_exec, C_undo)
RuntimeError: Operator bpy.ops.create.empty.poll() failed, context is incorrect

location: <unknown location>:-1

Using Blender 2.91
Code Pulled U20201221

**Comment by NevilArt** [comment]
Hi there
I test the Reference image create operator in default mode but I can`t see this error. can you explain a little bit more? when this happen? witch mode is active? and what is the view mode (perspective, orthographic, camera or ...) 

Thank you for report the issue.

**Comment by Finking4** [comment]
Hi,

Odd this morning its not erroring on creation anymore, this was in object mode, in perspective view. I was on 2.83 and updated yesterday to 2.91 to see if that would fix it so could of and then updated BSMAX so could of been looking at older files maybe. 

Now though once I set the image on the created object and try to resize it, by dragging on the yellow border. blender crashes. If I disable BSMAX its doesn't crash. I have attached the crash log

[file.crash.txt](https://github.com/NevilArt/BsMax_2_80/files/5734642/file.crash.txt)

Thanks for making this. its really helpful coming from learning Max in the past.
